tests/Context/Unit/Propagation/ArrayAccessGetterSetterTest.php

```diff
@@ -80,8 +80,12 @@ public function offsetExists($offset): bool
                 return true;
             }
 
-            public function offsetGet($offset)
+            /**
+             * @psalm-suppress ReservedWord
+             */
+            public function offsetGet($offset): mixed
```

Comment:
This might work, because the method is actually not called, still `mixed` is not a valid return type in PHP7.4, so can you please remove the stub methods by extending the class from `ArrayObject`, I guess this should work as well.
(Not sure what that interface is for anyway, since it's not used anywhere)

Reply:
replaced with a mock